### DISCUSSION TITLE
User report of corrupt database

### DISCUSSION THREAD

**Discussion by peppy** [realm/realm-dotnet#3461] [Q&A] [answered]
It's very rare that we hear of corruption issues, so I do wonder if the realm team would be interested in seeing one example reported to us by a user:

https://drive.google.com/file/d/1p-K3CCIsv8w6MQYkQMVygl9kO6wdhluK/view

If you don't have the bandwidth to look into this, that's fine. It's so rare that it might be due to a disk controller writing in a weird way, or something equally uncontrollable.

**Comment by nirinchev** [comment]
@jedelbo or @finnschiermer would that be interesting for you?

**Comment by jedelbo** [comment, marked as answer]
It looks like the file has been truncated.

**Reply by peppy** [reply]
Thanks for checking on it.

Is it safe to say that this is not a result of a realm-side operation? So either something we're doing or something the user (or their hardware) did externally?

**Comment by jedelbo** [comment]
Yes, we do not truncate files in that way.
